src/test/java/io/stargate/sgv2/jsonapi/testresource/StargateTestResource.java

```diff
@@ -159,25 +160,33 @@ public void stop() {
 
   private GenericContainer<?> baseCassandraContainer(boolean reuse) {
     String image = this.getCassandraImage();
+    String startupMessage =
+        this.isDse() ? ".*DSE startup complete.*\\n" : ".*Created default superuser role.*\\n";
```

Comment:
I would like to see if we can change this to be based on cqlsh access, as we have done in docker-compose files. We will probably need to come back and do that later as the initial version of the DSE 6.9 container we are currently testing with doesn't have cqlsh working yet.

Reply:
@jeffreyscarpenter Have to check how it can be done. Can that be done as separate PR?